CMakeLists.txt

```diff
@@ -455,17 +456,6 @@ if(${BLAS_INCLUDE_NEEDED} MATCHES ON )
 	install(FILES ${LAPACK_LIBRARIES} DESTINATION libs)
 endif(${BLAS_INCLUDE_NEEDED} MATCHES ON )
 
-# Kratos Testing. Install everything except sources
```

Comment:
why did u change this?

Reply:
![imagen](https://user-images.githubusercontent.com/19991680/50159136-3790bc80-02d6-11e9-99c8-7b881751b400.png)
I am not changing but the 14 files, please refresh your page